src/group/expand.rs

```diff
@@ -28,11 +28,13 @@ fn xor<L: ArrayLength<u8>>(x: GenericArray<u8, L>, y: GenericArray<u8, L>) -> Ge
 /// Corresponds to the expand_message_xmd() function defined in
 /// <https://www.ietf.org/archive/id/draft-irtf-cfrg-hash-to-curve-10.txt>
 pub fn expand_message_xmd<
+    'a,
     H: BlockInput + Digest,
     L: ArrayLength<u8>,
+    M: IntoIterator<Item = &'a [u8]>,
```

Comment:
I decided to pass through `[u8]`s instead of `u8`s to make sure that some not ideally optimized code wouldn't attempt to call `update()` on the hash inefficiently.